kubectl.sh create service crashes unhelpfully

I've reported this before.

```
$ ../../../cluster/kubectl.sh create -f serve-hostnames-service.yaml
Running: ../../../cluster/../cluster/gce/../../_output/dockerized/bin/linux/amd64/kubectl create -f serve-hostnames-service.yaml
F0209 17:34:34.126307   30570 create.go:78] 
```

From a Makefile I get exit status 255.
```
Running: ../../../cluster/../cluster/gce/../../_output/dockerized/bin/linux/amd64/kubectl create -f serve-hostnames-service.yaml
F0209 17:38:36.236419   31616 create.go:78] 
make: *** [create] Error 255
```

--- a/pkg/kubectl/cmd/create.go
+++ b/pkg/kubectl/cmd/create.go
@@ -57,6 +57,7 @@ Examples:
 				FilenameParam(flags.Filenames...).
 				Flatten().
 				Do()
+			checkErr(r.Err())
 
 			count := 0
 			err = r.Visit(func(info *resource.Info) error {

--- a/pkg/kubectl/cmd/delete.go
+++ b/pkg/kubectl/cmd/delete.go
@@ -71,6 +71,7 @@ Examples:
 				ResourceTypeOrNameArgs(args...).
 				Flatten().
 				Do()
+			checkErr(r.Err())
 
 			found := 0
 			err = r.IgnoreErrors(errors.IsNotFound).Visit(func(r *resource.Info) error {

--- a/pkg/kubectl/cmd/get.go
+++ b/pkg/kubectl/cmd/get.go
@@ -86,6 +86,7 @@ func RunGet(f *Factory, out io.Writer, cmd *cobra.Command, args []string) {
 			ResourceTypeOrNameArgs(args...).
 			SingleResourceType().
 			Do()
+		checkErr(r.Err())
 
 		mapping, err := r.ResourceMapping()
 		checkErr(err)

--- a/pkg/kubectl/cmd/update.go
+++ b/pkg/kubectl/cmd/update.go
@@ -60,6 +60,7 @@ Examples:
 				FilenameParam(flags.Filenames...).
 				Flatten().
 				Do()
+			checkErr(r.Err())
 
 			patch := cmdutil.GetFlagString(cmd, "patch")
 			if len(flags.Filenames) == 0 && len(patch) == 0 {

